collections/_infographics/emissions/emissions-eu-trends.md

```diff
@@ -13,4 +13,38 @@ data-orig:
+* __Residential, institutional and agricultural combustion:__ Emissions fell by 27% compared to 1990 to 519 million tonnes of CO<sub>2</sub>eq per year. These emissions are from heating and hot water (unless a heating plant supplies energy), cooking, etc. It also includes fuel combustion in agriculture and forestry.
+* __Combustion in industry and construction:__ Emissions fell by 41% to 429 million tonnes of CO<sub>2</sub>eq in 2019. Emissions come from the combustion of fossil fuels for industrial use, typically when something needs to be heated in production (during melting, drying, distillation, etc.).  
+For example, emissions from combustion in heavy industry (iron and steel production) have halved by 2019.
+* __Agriculture:__ Emissions from the agricultural sector have fallen by 21% since 1990 to 382 million tonnes of CO<sub>2</sub>eq per year. Emissions come mainly from livestock farming and soil cultivation and associated N<sub>2</sub>O emissions.  
+Reducing methane emissions from agriculture would lead to a reduction in the number of livestock kept (and the associated reduction in the consumption of beef and dairy products), a change in the management of manure (for example, by stabilising it in biogas plants) and less intensive fertiliser application. However, reducing livestock farming can also harm soil quality, availability of natural fertiliser, etc.  
+* __Waste management:__ Emissions from waste management have been declining since the mid-1990s. By 2019, they had fallen by 34% to 114 million tonnes of CO<sub>2</sub>eq per year. They are mainly produced by landfills, which leak methane into the atmosphere. This methane is produced by decomposing biodegradable material (paper, cardboard, textiles and bio-waste) in the landfill.
+* __Other:__ These are emissions not included in the previous categories. Emissions have been steadily declining - down 70% since 1990 to 8 million tonnes of CO<sub>2</sub>eq in 2019.
+
+## Notes on emissions data
+
+* The emission inventory provided by Eurostat uses the CRF (common reporting format) data format and structure. All calculation and reporting methodology is available on the website of the national greenhouse gas inventory programme ([NGGIP - national greenhouse gas inventory programme](https://www.ipcc-nggip.iges.or.jp/)) and is binding for all countries [UNFCCC](https://en.wikipedia.org/wiki/United_Nations_Framework_Convention_on_Climate_Change).
```

Comment:
change:
All calculation and reporting methodology is available on the website of the national greenhouse gas inventory programme 
to
All calculations and reporting methodologies are available on the website of the national greenhouse gas inventory programme

Reply:
Thanks for catching this mistake. I have updated the text to "calculations and reporting methodologies".